ansible/roles/vector/tasks/configure.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Install Vector configuration
-  template:
+  copy:
```

Comment:
This is to get around an issue where `template` tries to fill in the vector config's templates. We can probably get around it with escaping, but this seemed safer.

It also opens the question of, if we actually wanted to template some things in, how would we do it?

Reply:
Good point! We can use different `variable_end_string` and `variable_start_string` when resolving vector templates.

Reply:
Agree, @lukesteensen we use templates to insert addresses, ports, etc, so we'll need to revert this. I'll try to submit a PR today that changes this to use different `variable_end_string` and `variable_start_string` values.